### ISSUE TITLE
Torrent Downloads Tracker Failing in Sonarr and Radarr - Jackett Version 0.8.225.0

### ISSUE THREAD

**Issue by OasisFounder** [Jackett/Jackett#1902]
Receiving this error in Sonarr:

Invalid request Validation failed: 
 -- No results were returned from your indexer, please check your settings.

Error did not change by adding categories 5000,100008,5070,100001 as defined by Jacket categories,

================

Error is also present in Radarr - altering categories does not fix issue:

Invalid request Validation failed: 
 -- No results were returned from your indexer, please check your settings.

**Comment by garfield69** [comment]
try adding 2000 and 100004 to the list of categories in your Sonarr  TorrentDownloads Indexer definition  and see if that helps.

**Comment by OasisFounder** [comment]
Cannot seem to get this functional even with the change in categories

**Comment by garfield69** [comment]
on Sonarr I set up TorrentDownloads successfully with the following categories:      
`2000,3000,4000,4050,5000,5070,7000,8000`
![image](https://user-images.githubusercontent.com/6933240/31100711-031e5e40-a827-11e7-948d-491cef336976.png)

**Comment by OasisFounder** [comment]
I duplicated all of your settings and I am getting a failed with no results still.

**Comment by garfield69** [comment]
I'm sorry I don't know how to help you further.  Perhaps you need to restart Sonarr to reset any lingering too many errors/failures state?

**Comment by kaso17** [comment]
please provide the request ulr from the sonarr log

**Comment by OasisFounder** [comment]
From jackett side it states this in the log:

Found 0 (0 new) releases from Torrent Downloads

Using the below settings in Sonarr


![untitled-1](https://user-images.githubusercontent.com/30203057/31127148-06bf0e38-a81c-11e7-8a8a-209794a0d51b.jpg)

http://127.0.0.1:9117/api/v2.0/indexers/torrentdownloads/results/torznab/

Here's what its set on Jackett

![untitled-2](https://user-images.githubusercontent.com/30203057/31127512-2ae6a32e-a81d-11e7-99db-31f6a8c9d94b.jpg)


I dont have any problem clicking on the link and get to the torrentdownloads website

**Comment by kaso17** [comment]
Please check the sonarr log to get the exact URL (including arguments) used for testing.

**Comment by kaso17** [comment]
please also update to the latest jacket version.

**Comment by OasisFounder** [comment]
Ok - sorry took me a moment to find the log info that I think you're requesting. 

7-10-3 10:00:16.1|Debug|Torznab|Downloading Feed http://127.0.0.1:9117/api/v2.0/indexers/torrentdownloads/results/torznab/api?t=tvsearch&cat=2000,3000,4000,4050,5000,5070,7000,8000&extended=1&apikey=(removed)&offset=0&limit=100
17-10-3 10:00:18.5|Warn|NzbDroneErrorPipeline|Invalid request Validation failed: 
 -- No results were returned from your indexer, please check your settings.
17-10-3 10:00:18.5|Debug|Api|[POST] /api/indexer: 400.BadRequest (2471 ms)
17-10-3 10:00:30.0|Debug|Api|[GET] /api/health: 200.OK (4 ms)
17-10-3 10:00:30.4|Debug|Api|[GET] /api/health: 200.OK (3 ms)
17-10-3 10:00:31.1|Debug|Api|[GET] /api/diskspace: 200.OK (914 ms)
17-10-3 10:00:31.2|Debug|Api|[GET] /api/diskspace: 200.OK (699 ms)
17-10-3 10:00:32.5|Debug|Api|[GET] /api/log?page=1&pageSize=50&sortKey=time&sortDir=desc: 200.OK (13 ms)
17-10-3 10:00:32.6|Debug|Api|[GET] /api/log?page=1&pageSize=50&sortKey=time&sortDir=desc: 200.OK (10 ms)
17-10-3 10:00:47.7|Debug|Api|[GET] /api/log/file: 200.OK (2 ms)

**Comment by OasisFounder** [comment]
Jacket updated to Version 0.8.246.0 ; retested and still getting same error

**Comment by kaso17** [comment]
what happens if you open the link in your browser (insert the APIKEY again)

**Comment by OasisFounder** [comment]
I get offered to save a live bookmark:

![untitled-1](https://user-images.githubusercontent.com/30203057/31145612-3da1bdde-a852-11e7-9d70-1d3a1aefdd45.jpg)

**Comment by garfield69** [comment]
are you sure you pasted the link into the browser's address bar [and replaced the (removed) with your apikey] before pressing Enter ?

**Comment by OasisFounder** [comment]
Yes - that is what happened in Firefox; I cannot answer for how any other broswer would respond but I did change the (removed) api to the actual apikey

**Comment by garfield69** [comment]
Hang on a moment.       
I just looked back at your previous posts and see that your Jackett definition for torrentdownloads is an old one that asks for a username and password. This became unnecessary since v0.7.1719
So here is something you can try:       
go to the Jackett Dashboard, Trash the TorrentDownloads indexer entry and add it anew.       
Then try your test again.

**Comment by OasisFounder** [comment]
You might be on to something there ; I did delete the entry and re-added it ; but the new entry also has the username/password fields there.  And it still errors out the same way as before. 

I didn't really pay any attention to it originally because the logs were not suggesting a bad username/password - just no results.

**Comment by garfield69** [comment]
Hmm. Have you upgraded to the latest version of Jackett?      
Something you could try is:     
After deleting the entry, and before adding, try shutting down the Jackett service, and removing any leftover active definitions in the Jackett work-space.
On windows for example, the active Jackett torrentdownloads definition is stored in `C:\ProgramData\Jackett\Indexers\torrentdownloads.json`. 
And there is a *.bak as well. 
Then start everything up and add the entry.

**Comment by OasisFounder** [comment]
garfield69 - you have led me in the right direction. Here's something really odd - The Jackett torrentdownload indexer files were ok. But I deleted them and was still seeing a password field in the configuration. I started snooping around in the directories and found a file called "torrentDLME.yml" in the c:\programdata\Jacket\Definitions ; along with the file "torrentdownloads.yml". Both were configurations for that tracker but torrentDLME.yml was winning as the default definition file.

After I deleted that file ; everything worked as it should. I have no explanation as to how it ended up that way.
